apps/src/pythonlab/PythonlabView.tsx

```diff
@@ -70,15 +70,15 @@ const defaultConfig: ConfigType = {
   gridLayoutRows: '32px 232px auto',
   gridLayoutColumns: '300px auto',
   gridLayout: `
-    "instructions file-tabs"
-    "instructions editor"
-    "file-browser editor"
+    "instructions workspace"
+    "instructions workspace"
```

Comment:
I think we can clean this up now, by removing one of these duplicate lines. Above on line 70 we can just specify two rows. Instead of giving the rows a fixed height, we could try make it [flexible](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_grid_layout/Basic_concepts_of_grid_layout#the_fr_unit) like
```
gridLayoutRows: '1fr 1fr',
```
which would give each 50% of the available space. 

We could do similar improvements to the weblab2 layouts since they also have a couple of identical rows now.

Reply:
I might leave this for now since I'm going to work next on moving the console into Code Bridge, and this will get updated again.

Reply:
The duplicate line can safely be deleted, and the layoutRows can be simplified as well.

But also, not urgent especially if more layout changes are pending.

Reply:
I'm going to fast follow this with an update to both layouts!